### DISCUSSION TITLE
Problem uid.html parse failed

### DISCUSSION THREAD

**Discussion by RubiRoidbyZero** [nunocoracao/blowfish#1203] [Q&A] [answered]
Hi,
I'm new here, trying to figure out the blowfish template. I'm using Ubuntu version 22.04.3 LTS. I am installing it as per the instructions. After running the command "hugo server" I get the error:
"Module "blowfish" is not compatible with this Hugo version; run "hugo mod graph" for more information.
Error: add site dependencies: load resources: loading templates: "/home/user/hugo/themes/blowfish/layouts/partials/functions/uid.html:9:1": parse failed: template: partials/functions/uid.html:9: function "break" not defined"
If I delete this file, the server starts up normally. Is it important to keep this file in the directory?

**Comment by nunocoracao** [comment]
I wouldn't delete theme files. Seems like a good way to create more problems. 

Which version of the theme and hugo are you running. I would update everything and try again. Seems like one of those is not up to date.

**Reply by RubiRoidbyZero** [reply]
Hi Nuno,
hugo v0.92.2+extended linux/amd64 BuildDate=2023-01-31T11:11:57Z VendorInfo=ubuntu:0.92.2-1ubuntu0.1
theme 2.52.0

**Reply by nunocoracao** [reply]
Could you try updating Hugo?

**Reply by RubiRoidbyZero** [reply, marked as answer]
It was difficult, but I did it. Your edits did not help, there was the same error. Since I am new to Hugo and Ubuntu, I still do not understand all things well. Thanks to you, I discovered that I do not have the latest version of Hugo. I searched for a long time , how to upgrade it. In the end, I copied it directly from the repository. Now everything works well.
Nuno, thank you for your support.
